Hestia/app/src/main/java/com/rugged/application/hestia/Activator.java

```diff
@@ -0,0 +1,49 @@
+package com.rugged.application.hestia;
+
+public class Activator<T> {
+    private int activatorId;
+    private T state;
+    private String name;
+    private String stateType;
```

Comment:
I understand, this is something that needs to be coordinated and properly discussed with the server team, as Python is an OOP language, they are also in need of a better alternative.